### PR TITLE
Add crypto-square exercise

### PR DESCRIPTION
**Pull request by wasamasa**
Adapted from the Common Lisp track. I'm not 100% happy with my solution (I'd have preferred to either do string-only or buffer-only manipulation), but hey, maybe someone else will come up with a cooler one :)

### CODE REVIEW

File: exercises/crypto-square/crypto-square.el [hunk 1]
```diff
@@ -0,0 +1,11 @@
+;;; crypto-square.el --- Crypto Square (exercism)
+
+;;; Commentary:
+
+;;; Code:
+
+(require 'cl-lib)
```
Comment (benreyn) [inline exercises/crypto-square/crypto-square.el]: In the past I have tried to keep us from auto-including libraries in the solution template. I would like to not prime the student with a library inclusion and let them decide whether or not they would like to use any library code in their solution themselves.

File: exercises/crypto-square/example.el [hunk 2]
```diff
@@ -0,0 +1,32 @@
+(defun normalize-text (string)
```
Comment (benreyn) [inline exercises/crypto-square/example.el]: I think we should include the same commentary in the example solution as we do in the solution template.

Comment (wasamasa) [reply exercises/crypto-square/example.el]: Fixed

**Comment by benreyn** [comment]
@yurrriq, Want to give this one a second pair of eyes before I merge?

**Comment by yurrriq** [comment]
This example solution is a good starting place, and we can always improve it later. Thanks!

File: exercises/crypto-square/example.el [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+(defun chop-string (string columns)
+  (with-temp-buffer
+    (insert string)
+    (goto-char (point-min))
+    (while (not (eobp))
+      (ignore-errors (forward-char columns))
+      (insert "\n"))
+    (split-string (buffer-string) "\n" t)))
+
+(defun strings-column (strings column)
```
[the remark this answers is not in the document]
Comment (wasamasa) [reply exercises/crypto-square/example.el]: Fixed

**Pull request merged**